packages/editor/src/components/rich-text/patterns.js

```diff
@@ -68,7 +68,7 @@ export default function() {
 
 			const text = getTextContent( record );
 
-			if ( text.indexOf( '`' ) === -1 ) {
+			if ( text.includes( '`' ) ) {
```

Comment:
@tofumatt This is not the same. I guess it should have been `! text.includes( '`' )`. Additionally `includes` is not supported in IE11, so either we have to use lodash or revert.

Reply:
Oh, shoot, I'm sorry. I think we had an IE11 polyfill at Mozilla and I always forget about this one 😓